src/common/transformations/include/transformations/dimension_tracking.hpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+
+#include <transformations_visibility.hpp>
+
+#include <openvino/pass/graph_rewrite.hpp>
+#include <openvino/opsets/opset1.hpp>
+
+using namespace std;
+using P2Btype = std::unordered_map<ngraph::op::Parameter*, std::unordered_set<size_t>>;
+
+namespace ov {
+namespace pass {
+
+class TRANSFORMATIONS_API FindBatch;
```

Comment:
can this forward declaration be removed? FindBatch definition follows right after this declaration